catalystwan/workflows/config_migration.py

```diff
@@ -230,27 +217,80 @@ def collect_ux1_config(session: ManagerSession, progress: Callable[[str, int, in
     return ux1
 
 
-def push_ux2_config(session: ManagerSession, config: UX2Config) -> ConfigGroup:
-    """
-    Creates configuration group and pushes a UX2 configuration to the Cisco vManage.
-
-    Args:
-        session (ManagerSession): A valid Manager API session.
-        config (UX2Config): The UX2 configuration to push.
-
-    Returns:
-        UX2ConfigPushResult
-
-    Raises:
-        ManagerHTTPError: If the configuration cannot be pushed.
-    """
-
-    config_group_creator = ConfigGroupCreator(session, config, logger)
-    config_group = config_group_creator.create()
-    feature_profiles = config_group.profiles  # noqa: F841
-    for parcels in config.profile_parcels:
-        # TODO: Create API that supports parcel creation on feature profiles
-        # Example: session.api.parcels.create(parcels=parcels, feature_profiles=feature_profiles)
-        pass
-
-    return config_group
+def push_ux2_config(
+    session: ManagerSession, config: UX2Config, progress: Callable[[str, int, int], None] = log_progress
+) -> UX2ConfigRollback:
+    # Create mapping from origin ids
+    # do dataclass
+    # mapping = {
+    #     "config_group": {item.header.origin: item for item in config.config_groups},
+    #     "feature_profile": {item.header.origin: item for item in config.feature_profiles},
+    #     "profile_parcels": {item.header.origin: item for item in config.profile_parcels},
+    # }
+    # rollback_config_groups_ids: List[UUID] = []
+    # rollback_feature_profiles_ids: List[Tuple[UUID, Literal["system", "other", "transport"]]] = []
+
+    config_pusher = UX2ConfigPusher(session, config)
+    rollback = config_pusher.push()
+    # for config_group in config.config_groups:
```

Comment:
Can we remove this commented code already?

Reply:
Removed commented code.